### DISCUSSION TITLE
How to disable eager loading?

### DISCUSSION THREAD

**Discussion by adityathebe** [gatsbyjs/gatsby#37531] [Help] [answered]
I'm using Gatsby `v2.32.4`.

I've recently migrated my website from Netlify to self hosting with Nginx. I want to know on which page a viewer currently is using nginx logs. However, the problem for me is that when a user visits my home page gatsby sends `page-data/*.json` request to all my pages. This makes it virtually impossible to get an accurate analytics.

Is it possible to prevent this so that a request to a page is only made when the user actually visits?

I understand this makes my website slightly slower to navigate, but I want to experiment with it.

**Comment by violy** [comment, marked as answer]
Hello !
Basically, you could prevent the behavior of Gatsby Link Api using the basic `<a>` anchor tag instead of `<Link>` – and `window.location.href=` instead of `navigate()` … but you loose all the fluidity and preload mechanisms. 

Why not use a dedicated analytics framework, like Matomo or GA ?

**Reply by adityathebe** [reply]
Thanks!

> Why not use a dedicated analytics framework, like Matomo or GA

I do have [Simple Analytics](https://simpleanalytics.com) set up. I'm just trying to dabble around with things right now.
